recipes/liblbfgs/meta.yaml

```
@@ -0,0 +1,48 @@
+{% set name = "liblbfgs" %}
+{% set version = "1.10" %}
+{% set sha = "4158ab7402b573e5c69d5f6b03c973047a91e16ca5737d3347e3af9c906868cf" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/downloads/chokkan/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha }}
+
+build:
+  number: 0
+  run_exports:
+  - {{ pin_subpackage('liblbfgs', max_pin='x.x') }}
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+
+test:
+  commands:
+    - test -f ${PREFIX}/include/lbfgs.h
```

These will need to have windows specific test cases like:
```suggestion
    - test -f ${PREFIX}/include/lbfgs.h  # [not win]
    - if not exist %LIBRARY_INC%/lbfgs.h exit 1  # [win]
```

---

I guess also the others?

---

Yup, though I'm not very familiar with Windows and not sure what the paths will be.

The [`json-c`](https://github.com/conda-forge/json-c-feedstock/blob/master/recipe/meta.yaml#L29-L32) and [`ldas-tools-cmake`](https://github.com/conda-forge/ldas-tools-cmake-feedstock/blob/master/recipe/meta.yaml#L26-L30) feedstocks look like they do something similar.

---

OK. But I don't expect the library to build on windows anyway, do you. The configure, make, make install seems pretty Linux specific.

---

Ah sorry I missed that you aren't building this for windows (it would use a `bld.bat` file if one existed). If you want to skip Windows entirely you should add `skip: True  # [win]` to the `build` section.